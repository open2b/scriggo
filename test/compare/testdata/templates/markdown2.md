{# render #}

<a href="{{ `"` }}">

<script>var v = {{ "a" }};</script>

<style>div { background: url(data:image/png;base64,{{ []byte("\x12\x34\x89") }}) }</style>

<![CDATA[ {{ ".5" }} ]]>
