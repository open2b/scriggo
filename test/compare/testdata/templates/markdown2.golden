
<a href="%22">

<script>var v = "a";</script>

<style>div { background: url(data:image/png;base64,EjSJ) }</style>

<![CDATA[ \.5 ]]>

    *** spaces code block ***
	*** tab code block ***

\{{ 1 }\}

\<a b="""></a>

<c d="#d">\#e</c>

\#a <!-- #a -->  \#a \<b\>\#b\</b\>  \#a
